### DISCUSSION TITLE
hht_alpha  value

### DISCUSSION THREAD

**Discussion by selarem** [idaholab/moose#26257] [Q&A Modules: Solid mechanics]
Hello,
I think as per moose formulation of dynamics problem, the hht_alpha should be  -1/3 <=hht_alpha<=0 (Géradin Rixen book). However, In the examples I saw :
```
[Kernels]
  [./DynamicTensorMechanics]
    displacements = 'disp_x disp_y disp_z'
    hht_alpha = -0.3
    stiffness_damping_coefficient = 0.1
  [../]
```
but also, I saw:
```
[Kernels]
  [DynamicTensorMechanics]
    displacements = 'disp_x disp_y disp_z'
    stiffness_damping_coefficient = 0.1
    hht_alpha = 0.11
  []
```
I would like to know if there is any reason for this positive value.

Regards,
Saber

**Comment by maxnezdyur** [comment]
No use the negative only values.

In `ADInertialForceShell` there is a check to make sure that it is negative but that doesn't exist for AD/InertialForce.
```
 params.addRangeCheckedParam<Real>("alpha",
                                    0.0,
                                    "alpha >= -0.3333 & alpha <= 0.0",
                                    "Alpha parameter for mass dependent numerical damping induced "
                                    "by HHT time integration scheme");
```

**Reply by GiudGiud** [reply]
@maxnezdyur should we be erroring in the tensor mecahnics action with alpha > 0 then?

**Reply by maxnezdyur** [reply]
Yes I think so. I think the error check never made it's way there.
